src/zeek.bif

```diff
@@ -2675,12 +2675,12 @@ function double_to_count%(d: double%): count
 ##          an invalid format.
 ##
 ## .. zeek:see:: to_addr to_int to_port to_subnet
-function to_count%(str: string%): count
+function to_count%(str: string, base: count &default=10%): count
 	%{
 	const char* s = str->CheckString();
 	char* end_s;
 
-	uint64_t u = (uint64_t) strtoull(s, &end_s, 10);
+	uint64_t u = (uint64_t) strtoull(s, &end_s, base);
```

Comment:
Nit: since we are touching this already we could fix this to use a proper cast, e.g.,

```suggestion
	uint64_t u = static_cast<uint64_t>(strtoull(s, &end_s, base));
```